### DISCUSSION TITLE
Allow clearing specific mail entry's

### DISCUSSION THREAD

**Discussion by Josh65-2201** [EssentialsX/Essentials#5100] [Ideas and Feature Suggestions] [answered]
Currently I can only clear all mail when the read commands used. Its annoying to manage the mail boxs like that as old ones build up without been able to be cleared.

**Comment by JRoy** [comment, marked as answer]
you can clear individual mails by doing `/mail clear 1` `/mail clear 2` `/mail clear 3` etc

**Reply by Josh65-2201** [reply]
Thanks, thought I tried that. Could it be made so that tab complete has a number entry or say in the chat when it prompts /mail clear that numbers can be used.
